pom.xml

```diff
@@ -31,14 +31,9 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>6.0.4</version>
+            <version>6.1.0</version>
             <optional>true</optional>
         </dependency>
-        <dependency> <!-- TODO JENKINS-33095 workaround -->
-            <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
-            <artifactId>icon-shim</artifactId>
```

Comment:
I am afraid I do not understand the policy around `icon-shim` at all. Cf. https://github.com/jenkinsci/plugin-pom/pull/72.